Open Judge System/Workers/OJS.Workers.Executors/RestrictedProcessExecutor.cs

```diff
@@ -191,6 +191,13 @@ public RestrictedProcessExecutor(int baseTimeUsed, int baseMemoryUsed)
             {
                 result.Type = ProcessExecutionResultType.RunTimeError;
             }
+            else
+            {
+                if (result.ExitCode != 0 && result.ExitCode != -1)
```

Comment:
just add this check litteraly in the above `if(..)` with `||` no need of `else` statement